prqlc/crates/compiler/README.md

```diff
@@ -1,17 +1,18 @@
 # PRQL compiler
 
-`prql-compiler` contains the implementation of PRQL's compiler, written in Rust.
+`prqlc` is the reference implementation of a compiler from PRQL to SQL, written
+in Rust.
 
 For more on PRQL, check out the [PRQL website](https://prql-lang.org) or the
 [PRQL repo](https://github.com/PRQL/prql).
 
 For more usage examples and the library documentation, check out the
-[`prql-compiler` documentation](https://docs.rs/prql-compiler/latest/prql_compiler/).
+[`prqlc_main` documentation](https://docs.rs/prqlc_main/latest/prql_compiler/).
```

Comment:
What's `prqlc_main`?

Reply:
I had prql-compiler renamed to `prqlc/crates/main`.